feature-policy/META.yml

```
@@ -1,3 +1,2 @@
 spec: https://wicg.github.io/feature-policy/
 suggested_reviewers:
```

As confirmed in the main comments, you can apply the following suggestion: 
 
```suggestion
 suggested_reviewers:
  - bakulf 
```